src/vs/workbench/browser/quickopen.ts

```diff
@@ -250,35 +256,53 @@ export class EditorQuickOpenEntry extends QuickOpenEntry implements IEditorQuick
 	public run(mode: Mode, context: IEntryRunContext): boolean {
 		const hideWidget = (mode === Mode.OPEN);
 
+		let sideBySide;
 		if (mode === Mode.OPEN || mode === Mode.OPEN_IN_BACKGROUND) {
-			let sideBySide = context.keymods.indexOf(KeyMod.CtrlCmd) >= 0;
-
-			let openInBackgroundOptions: IEditorOptions;
-			if (mode === Mode.OPEN_IN_BACKGROUND) {
-				openInBackgroundOptions = { pinned: true, preserveFocus: true };
-			}
+			sideBySide = !context.quickNavigateConfiguration && context.keymods.indexOf(KeyMod.CtrlCmd) >= 0;
+		}
 
-			let input = this.getInput();
-			if (input instanceof EditorInput) {
-				let opts = this.getOptions();
-				if (opts) {
-					opts = objects.mixin(opts, openInBackgroundOptions, true);
-				} else if (openInBackgroundOptions) {
-					opts = EditorOptions.create(openInBackgroundOptions);
-				}
+		let pinned;
```

Comment:
@wprater why do we need pinned here? first of all it does not seem to be used when the mode is `Mode.OPEN`. And then I think the editor part itself is already dealing with the setting `workbench.editor.enablePreviewFromQuickOpen`?